plugin/trino-base-jdbc/pom.xml

```diff
@@ -83,6 +83,12 @@
             <artifactId>units</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.opentelemetry.instrumentation</groupId>
+            <artifactId>opentelemetry-jdbc</artifactId>
+            <version>1.29.0-alpha</version>
```

Comment:
I'm not sure whether this version isn't already imported from bom. Can you check that?